src/components/Transactions/TransactionContract.tsx

```diff
@@ -31,6 +32,12 @@ export const TransactionContract = ({ txDetails }: { txDetails: TxDetails }) =>
     }
   })()
 
+  const isReceive = txDetails.tradeTx?.type === TxType.Receive
+  const isWithdraw = txDetails.tx.data?.method === ContractMethods.Withdraw
```

Comment:
The naming here is a little off - this is more like `interactsWithWithdrawMethod`.